SQRLUtilsLib/SQRLCPSServer.cs

```diff
@@ -205,5 +257,56 @@ private byte[] GenerateGif()
             const string clearGif1X1 = "R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==";
             return Convert.FromBase64String(clearGif1X1);
         }
+
+
+        /// <summary>
+        /// Handles CPS response for all requests
+        /// </summary>
+        /// <param name="succesUrl">Success URL passed in if the response is a successful Auth Ident</param>
```

Comment:
Maybe document the other parameters, too?

Reply:
ah yes I added those later LoL on it!